src/libslic3r/PrintConfig.hpp

```diff
@@ -729,6 +729,7 @@ PRINT_CONFIG_CLASS_DEFINE(
     // Overhang angle threshold.
     ((ConfigOptionInt,                 support_threshold_angle))
     ((ConfigOptionFloat,               support_object_xy_distance))
+    ((ConfigOptionFloat,               support_object_base_additional_xy_distance))
```

Comment:
It would be better to define it as parameter: support_object_top_interface_xy_distance
So top contacts us this parameter while other support layers use `support_object_xy_distance`